### ISSUE TITLE
[Get Familiar][Non core] Submit your sample image and template!

### ISSUE THREAD

**Issue by Udayraj123** [Udayraj123/OMRChecker#40]
As you setup OMRChecker and run samples according to the Readme, you would maybe want to try it on new public images or some of your own. This issue is created to accept new submissions. This is your chance to feature your template and image in the [samples section](https://github.com/Udayraj123/OMRChecker/tree/master/samples)!

Steps to follow - 
1. Pick any OMR sheet image(s) that you can use publicly.
2. Design a [template overlay](https://github.com/Udayraj123/OMRChecker/wiki/About-Templates) for the image(using `--setLayout` flag). Discuss on discord for any issues faced.
3. Make sure to see it generates correct output. Put it in samples folder and raise a PR.

**Note:** this issue is marked with the `hacktoberfest` label. Follow `#hacktoberfest-discussions` [on Discord](https://discord.gg/ANUuzsrpVV) for further details .

**Comment by issac5693** [comment]
should the sample image be a omr one ad what is a template

**Comment by Udayraj123** [comment]
@issac5693 Yes it should look like any of the existing samples. 
> what is a template
A template.json file is needed for OMRChecker to identify the bubble positions to use (see image)
<img width="193" alt="image" src="https://user-images.githubusercontent.com/16881051/193246935-c6258c30-24ad-4977-bc1b-519ab470d2c2.png">


Checkout [User Guide](https://github.com/Udayraj123/OMRChecker/wiki/User-Guide) for more details

You can find both template json files and sample images here:
https://github.com/Udayraj123/OMRChecker/tree/master/samples

**Comment by Antibodyy** [comment]
Hey. Can I take up this issue?

**Comment by Udayraj123** [comment]
Hi @Antibodyy can you create a PR or a draft PR for this? Then I can assign it to you. Hope you've started a discussion discord as well

**Comment by Sandeep-1507** [comment]
@Udayraj123 I want to try this same with a sample NEET 2022 OMR sheet,it can be shown publicaly.Should I do it?

**Comment by Udayraj123** [comment]
@Sandeep-1507 that would be great actually! As long as you can reference its source where you got the image from. We should be fine. Also we can avoid showing any student name, if present we can blur it.

**Comment by Udayraj123** [comment]
linking https://github.com/Udayraj123/OMRChecker/pull/64

**Comment by Udayraj123** [comment]
yes can do! just make sure there's some new varieties

**Comment by Udayraj123** [comment]
Added ideas for different types of OMR sheets.

**Comment by amankumar003** [comment]
hello @Udayraj123 I want to add my contribution to this repo by adding contribution to this my first issue

**Comment by Udayraj123** [comment]
Sure you can go ahead. Prepare a template for a new OMR Sheet

**Comment by Udayraj123** [comment]
@ShamanthVallem I saw your PR, I think it can qualify just go through Contributing.md

**Comment by ShamanthVallem** [comment]
@Udayraj123 please review my PR #108, I updated the template.json file

**Comment by RohanHBTU** [comment]
@Udayraj123 please review my PR #110 and share the required improvements.

**Comment by Udayraj123** [comment]
In case anyone wants to work on a new template - check this Survey template: https://www.cse.iitd.ac.in/~aseth/udai/OMRProj/test/2circle-3-small.gif

**Comment by sunnanxuan** [comment]
Hello ,
I am new developer and looking to do my 1st contribution here. I came across this issue and am very interested in contributing to the project by providing the requested sample image. I hope I can use this issue to help me familiar with the process to solve issues on github with fork, clone, commit, push and pull request

**Comment by Udayraj123** [comment]
Hello @sunnanxuan, welcome to the OMRChecker community! 
Sure, try out the instructions in Readme and share your sample once you get the setup ready. Feel free to join the discord channel for quicker updates :)

**Comment by Udayraj123** [comment]
Adding a link to a few public images of OMR sheets : 
- https://www.kishanganjbihar.com/2012/05/amu-mbbs-bds-2012-answer-key.html
- https://www.facebook.com/profile/100068874830284/search/?q=answer%20key&locale=mt_MT

**Comment by Udayraj123** [comment]
Hey @Dxuian, yes it's a continuous issue i.e. 1 PR per sample. We're still looking for a variety of samples to be added. 
If you pick this up, please make sure the image size is compressed and under 1500 pixels in width.

**Comment by Dxuian** [comment]
ive added #227  please review

**Comment by samuelIkoli** [comment]
@Udayraj123 I made a PR for a sample document, please review 🙏 .

**Comment by Udayraj123** [comment]
Hey @samuelIkoli, thanks for the submission. You may check my [review comments](https://github.com/Udayraj123/OMRChecker/pull/232#pullrequestreview-2396917447)

**Comment by samuelIkoli** [comment]
> [Feature] Support for sortType and sortOrder in outputColumns #209

Thank you very much for the reply, I have checked your review comments and worked on them.

**Comment by Rugvedrc** [comment]
Hi @Udayraj123, I'd like to contribute a sample OMR sheet for this issue. Could you please assign it to me? I'm thinking of adding a JEE/NEET sample as mentioned in the issue.

**Comment by Udayraj123** [comment]
@Rugvedrc sure, please check existing samples(I think there's one for NEET and JEE for some year, but a different format is still appreciated) note that the image has taken by you and/or is eligible to be public.

**Comment by Rugvedrc** [comment]
> @Rugvedrc sure, please check existing samples(I think there's one for NEET and JEE for some year, but a different format is still appreciated) note that the image has taken by you and/or is eligible to be public.

Thank you @Udayraj123! I'll check the existing samples first and find a different format. I'll set up OMRChecker locally and create the template. Will open a draft PR once ready.

**Comment by Rugvedrc** [comment]
Hey @Udayraj123! 

I've added a NEET OMR sample for 180 questions.

PR: #287

Details:
- Blank NEET OMR sheet (1238x1688px)
- 5 columns × 36 questions each
- QTYPE_MCQ4 format (options 1-2-3-4)
- Verified zero false positives on blank sheet

Please review when you get a chance!

**Comment by abhi-yanta** [comment]
Hi @Udayraj123 — I'm a new developer looking to make my first open-source contribution. I came across this issue and am interested in working on it. Could you let me know if it's still available to pick up? I'd like to use it to learn the full contribution flow (fork → clone → branch → commit → PR). Happy to share my approach before starting, as you've asked others to do. Thanks!

**Comment by Udayraj123** [comment]
@abhi-yanta, yes this ticket is tagged `continuous` so it'll be available for new contributions. Please check existing PRs that have referenced this issue to understand how to submit a new sample with test cases.
